src/main/webapp/customer-center/new-subscription.jsp

```diff
@@ -18,73 +20,153 @@
 <%@include file="../fragments/navbar.jsp" %>
 
 <div class="container-fluid bg-1 text-center">
-    <img src="${pageContext.request.contextPath}/images/logo-big.png" class="img-responsive img-circle" style="display:inline;" width="150" height="150">
+    <img src="${pageContext.request.contextPath}/images/logo-big.png" class="img-responsive img-circle"
+         style="display:inline;" width="150" height="150">
     <h2>Új előfizetés</h2>
 </div>
 
+<script>
+
+    function chooseProduct(id, quantity) {
+        $.post(
+            '${pageContext.request.contextPath}/customer-center/select-product',
+            {id: id, quantity: quantity},
+            (data, status) => {
+
+            }
+        )
+    }
+</script>
```

Comment:
Ez a `<script>` nem tudom mit csinál itt, erre nincsen szükség.
Volt egy ötletem, hogy mutatok nektek egy kis `ajax`-ot, de ez már nagyon frontend specifikus lenne. Egyébként ezzel megvalósítható lenne JS-sel küldeni kéréseket a szervernek és nem `<form>`-on keresztül. Az előbbi megközelítés a modern frontend alappillére, de még elő fordulhat néha az a kérés küldési módszer is, amit mi alkalmaztunk órán (de nem túl modern, de nagyon jól mutatja be a web működésének alapjait).